admin_guide/scheduling/scheduler-advanced.adoc

```diff
@@ -0,0 +1,53 @@
++
+Pod affinity uses labels on nodes and label selectors on pods to create rules for pod placement. Rules can be mandatory (required) or best-effort (preferred).
++
+See xref:../../admin_guide/advanced_scheduling/pod_affinity.adoc#admin-guide-sched-pod-affinity[Using Pod Affinity and Anti-Affinity].
+
+[[admin-guide-scheduling-advanced-node]]
+Node Affinity:: Node affinity allows a *pod* to specify an affinity (or anti-affinity) towards a group of *nodes* (due to their special hardware, location, requirements for high availability, and so forth) it can be placed on.  The node does not have control over the placement. 
++
+Node affinity uses labels on nodes and label selectors on pods to create rules for pod placement. Rules can be mandatory (required) or best-effort (preferred).
++
+See xref:../../admin_guide/advanced_scheduling/node_affinity.adoc#admin-guide-sched-affinity[Using Node Affinity].
+
+[[admin-guide-scheduling-advanced-selector]]
+Node Selectors:: Node selectors are the simplest form of advanced scheduling. Like node affinity, it also uses labels on nodes and label selectors on pods to allows a *pod* to control the *nodes* on which is can be placed. However, node selectors do not have required and preferred rules that node affinities have.
++
+See xref:../../admin_guide/advanced_scheduling/node_selector.adoc#admin-guide-sched-selector[Using Node Selectors].
+
+[[admin-guide-scheduling-advanced-taints]]
+Taints and Tolerations:: Taints/Tolerations allow the *node* to control which *pods* should (or should not) be scheduled on them. Taints are labels on a node and tolerations are labels on a pod. The labels on the pod must match (or tolerate) the label (taint) on the node in order to be scheduled. 
++
+Taints/tolerations have one advantage over affinities. For example, if you add to a cluster a new group of nodes with different labels, you would need to update affinities on each of the pods you want to access the node and on any other pods you do not want to use the new nodes. With taints/tolerations, you would only need to update those pods that are required to land on those new nodes, because other pods would be repelled.  
++
+See xref:../../admin_guide/advanced_scheduling/taints_tolerations.adoc#admin-guide-taints[Using Taints and Tolerations].
+
+== Using Multiple Methods
+
+These different methods can be used together to create the most effificent processes for your environment. For example, if you have a dedicated group of nodes and want specific pods to be placed only on those nodes, you can uses a combination of taints/tolerations and node affinity to make sure that appropriate new pods are scheduled on those nodes and prevent other new pods from being scheduled on those nodes. 
```

Comment:
@gaurav-nelson @wmengRH "Verify" is not correct here. But, "make sure" might not be the best wording. It should be "to ensure that appropriate new pods are scheduled..."
I used this style guide as a reference (see "ensure" in Chapter 13),